bounty_agent.py

```diff
@@ -123,18 +125,20 @@ def job(self) -> None:
 
     def job_listener(self, event):
         if event.exception:
-            self.logger.info('The job failed')
+            self.logger.info('"Get Bounty" job failed')
             utc_now = datetime.utcnow()
-            self.scheduler.add_job(self.job, 'date', run_date=utc_now + timedelta(seconds=60))
+            self.scheduler.add_job(self.job,
+                                   'date',
+                                   run_date=utc_now + timedelta(seconds=DELAY_AFTER_ERR))
             self.logger.debug(self.scheduler.get_jobs())
         else:
-            self.logger.debug('The job finished successfully)')
-            reward_date = self.get_reward_date()
-            self.logger.info(f'Next reward date after job: {reward_date}')
-            utc_now = datetime.utcnow()
-            if utc_now > reward_date:
-                self.logger.debug('Changing reward time by current time')
-                reward_date = utc_now
+            self.logger.debug('"Get Bounty" job finished successfully)')
+            try:
+                reward_date = self.get_reward_date()
+                self.logger.info(f'Next reward date after getBounty job: {reward_date}')
+            except Exception:
+                reward_date = datetime.utcnow() + timedelta(seconds=DELAY_AFTER_ERR)
```

Comment:
It's better to log actual exception.

Reply:
this exception is already logged in `get_reward_date()` function (along with telegram notification)